### ISSUE TITLE
Allow non-spammy links in CKAN profiles

### ISSUE THREAD

**Issue by martinlugton** [ckan/ckan#4733]
### CKAN Version if known (or site URL)

On pages like...
https://ckan.publishing.service.gov.uk/user/edit/nmtoken
[Where nmtoken is the username]

### Please describe the expected behaviour

User is able to include non-spammy URLs in their profile

### Please describe the actual behaviour

URLs are blocked

### What steps can be taken to reproduce the issue? 

Try to add a URL to a user's profile

### Why this is worth addressing

Spammy URLs in profiles are bad, but legitimate URLs in profiles could be of benefit.
In one example case, the user wanted to link to their orcid identifier.

**Comment by smotornyuk** [comment]
By the way, currently, you can paste link into profile if no schema provided, for example ```[Link to Google](//google.com)```. Obviously, if your target site uses a different protocol, it won't work, thou

**Comment by amercader** [comment]
@martinlugton I agree in principle, but how can you distinguish spammy from non-spammy links? :)

This limitation should be lifted if the maintainers of the site decide so but sadly it's a bit difficult now as there is no direct way to extend or modify the user schema from an extension in the same way datasets or orgs. A config option to allow links in profiles could be an easier approach.
